mypy.ini

```diff
@@ -38,7 +38,7 @@ files =
     test/test_type_info.py,
     test/test_utils.py,
     tools/linter/clang_format_utils.py,
-    tools/linter/clang_tidy.py,
+    tools/linter/clang_tidy,
     tools/generate_torch_version.py,
     tools/render_junit.py,
     tools/stats
```

Comment:
we can probably just remove all of these lines since all of `tools/` is already included in `mypy-strict.ini`, right?